src/components/News.js

```diff
@@ -13,14 +12,26 @@ class News extends Component {
     this.setState({ isEditModeOn: !this.state.isEditModeOn });
   };
 
+  isAuthor = () => {
+    if (this.props.currentUser.userId === this.props.news.entry.userId) {
```

Comment:
Can be simplified to `return this.props.currentUser.userId === this.props.news.entry.userId`